### ISSUE TITLE
tqdm progressbar output messy in documentation

### ISSUE THREAD

**Issue by OnnoEbbens** [gwmod/nlmod#312]
The documentation looks kind of messy because intermediate status of the progress bar is shown:

![image](https://github.com/gwmod/nlmod/assets/10562903/6fb9a0d5-40d6-48c3-9029-0775d6498892)

A fix might be to disable the progress bar when creating the documentation by setting an environment variable with `TQDM_DISABLE=1`, see also: https://github.com/tqdm/tqdm/pull/1061

**Comment by rubencalje** [comment]
Hmm, this used to be prettier:

<img width="528" alt="image" src="https://github.com/gwmod/nlmod/assets/18527362/41f9f937-7a52-48a4-8494-8a9b31eec4c5">

What did we change?

**Comment by dbrakenhoff** [comment]
Maybe I changed `from tqdm import tqdm` to `from tqdm.auto import tqdm`?

But the latter option does make things much prettier for users running notebooks, with less red highlighting and such.

**Comment by rubencalje** [comment]
> Maybe I changed `from tqdm import tqdm` to `from tqdm.auto import tqdm`?
> 
> But the latter option does make things much prettier for users running notebooks, with less red highlighting and such.

I do not think that is the problem, as we still use `from tqdm import tqdm` I think. But it may be the solution?

**Comment by rubencalje** [comment]
Another import-method is no solution, I found out from testing. I think the problem has to do with the nbconvert version. Since nbconvert version 7.14 the output is messy. See https://github.com/jupyter/nbconvert/issues/2092

I will fix the nbsphinx version to 7.13 for now, and hope this will make thoe output pretty again,

**Comment by rubencalje** [comment]
Adding `ipywidgets` to the dependencies was also necessary.

I also made these two small changes to the main-branch, as I hoped this would fix the stable documentation as well. This is not the case, however. Then I made a branch named 'stable', which according to some webpages would fix it. This is also not the case. Finally, I temporarily set the default docs-page to the latest-version, from stable. We should change this back after the next release.
